test/conftest.py

```diff
@@ -1,4 +1,11 @@
 import os
+
+# Disable any AWS endpoint-url environment variables overrides for testing.
+if "S3_URL" in os.environ:
+    os.environ.pop("S3_URL")
+if "SQS_URL" in os.environ:
+    os.environ.pop("SQS_URL")
```

Comment:
Does this not globally eliminate these values? You probably want a session scoped fixture that pops these values back on and off, just in case? Though probably not a big deal in local environment.

Reply:
Yeah, I was a little unsure of this - was doing this because when I had this env vars set in my env, because I was using this facility for testing locally with localstack, tests were failing (because tests not setup of course with the assumption of using localstack), so I just globally unset these variables for all testing.